pandas/core/groupby/generic.py

```diff
@@ -2470,14 +2521,76 @@ def take(
         result = self._op_via_apply("take", indices=indices, axis=axis, **kwargs)
         return result
 
-    @doc(DataFrame.skew.__doc__)
     def skew(
         self,
         axis: Axis | None | lib.NoDefault = lib.no_default,
         skipna: bool = True,
         numeric_only: bool = False,
         **kwargs,
     ) -> DataFrame:
+        """
+        Return unbiased skew within groups.
+
+        Normalized by N-1.
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns', None}, default 0
+            Axis for the function to be applied on.
+
+            For DataFrames, specifying ``axis=None`` will apply the aggregation
```

Comment:
Can remove "For DataFrames" here - that is used when the docstring is shared between Series and DataFrame implementation.

Reply:
Actually - it looks like axis=None here is buggy:

```
df = pd.DataFrame({'a': [1, 1, 1, 1], 'b': [2, 3, 4, 20], 'c': [3, 4, 5, -20]})
print(df[['b', 'c']].skew(axis=None))
# -0.9424450936999188

gb = df.groupby('a')
result = gb.skew(axis=None)
print(result)
#           b         c
# a                    
# 1  1.945331 -1.972398
```

It looks like the implementation of _op_via_apply does not correctly handle this. Could you open an issue if there isn't one?

Reply:
Done (see below).